project/config/lgbcni/config/core.entity_form_display.node.consultation_response.default.yml

```diff
@@ -0,0 +1,107 @@
+uuid: eaaaefed-4441-4220-8e39-3ce7065ad24e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.consultation_response.field_documents
+    - field.field.node.consultation_response.field_response_type
+    - field.field.node.consultation_response.field_stage
+    - node.type.consultation_response
+  module:
+    - content_moderation
+    - file
+    - path
+id: node.consultation_response.default
+targetEntityType: node
+bundle: consultation_response
+mode: default
+content:
+  created:
+    type: datetime_timestamp
+    weight: 10
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  field_documents:
```

Comment:
It is good to keep the naming convention the same for Unity field_attachment and for it to be an entity reference field.